raptorWeb/templates/raptormc/base.html

```diff
@@ -99,7 +99,11 @@ <h5 class="offcanvas-title text-center" id="offcanvasNavLabel">Navigation</h5>
           <div class="navbar-header d-flex">
             <div class="row p-2 pt-1 pb-1">
               <a class="col-3 col-sm-2 col-lg-1 btn btn-outline-secondary" role="button" data-bs-toggle="tooltip" 
+              {% if discord_guild %}
               data-bs-title="{{ discord_guild.online_members }}/{{ discord_guild.total_members }} Members Online" href="https://discord.gg/GEhyhku" target="_blank">
+              {% else %}
+              data-bs-title="No Guld" href="https://discord.com" target="_blank">
```

Comment:
Typo, and this can probably be better represented when the Guild is not present.

Reply:
I will do a lot more with the bot in a future PR, possibly generating and displaying invite links in that button if the bot is enabled/not showing it at all if not. This change was out of scope for this PR in general.